packages/devtools-kit/src/state/state.ts

```diff
@@ -1,3 +1,4 @@
+import { ref } from 'vue'
```

Comment:
Maybe we should find a better way to handle it to avoid importing the vue library.